packages/sabnzbd/build.sh

```diff
@@ -0,0 +1,31 @@
+TERMUX_PKG_HOMEPAGE=https://sabnzbd.org/
+TERMUX_PKG_DESCRIPTION="Fully automated Usenet Binary Downloader"
+TERMUX_PKG_LICENSE="GPL-2.0, GPL-3.0"
+TERMUX_PKG_LICENSE_FILE="LICENSE.txt, GPL2.txt, GPL3.txt"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=3.7.2
+TERMUX_PKG_SRCURL=https://github.com/sabnzbd/sabnzbd/releases/download/${TERMUX_PKG_VERSION}/SABnzbd-${TERMUX_PKG_VERSION}-src.tar.gz
+TERMUX_PKG_SHA256=01250ecd09cdb50da8be8829e4f44f33f0bf1bcf1fcff136ad6045e5f4a2a172
+TERMUX_PKG_DEPENDS="python (>= 3.7.1), python-cryptography, python-sabyenc3 (>= 5.4.4), par2, unrar, p7zip, unzip"
```

Comment:
I think we can drop the (>= versions), generally we just use those when we fix issues. In this case there is no python-sabyenc3 < 5.4.4, and python has been >= 3.7.1 for a loong time